config.js

```diff
@@ -89,6 +89,9 @@ var config = {
     // Disables self-view tile. (hides it from tile view and from filmstrip)
     // disableSelfView: false,
```

Comment:
Can you also rename this to hideSelfView so that it's more clear?

Reply:
We had pushed already as stable with it ... so I do not want to rename it ...